.gitignore

```diff
@@ -1,2 +1,4 @@
 /vendor
 /composer.phar
+/.idea/
+.gitignore
```

Comment:
I'm fine with .idea being removed (I recognize that directory as being one PHP Storm creates) but I'll probably do another commit after I merge this one removing .gitignore. For ignoring .gitignore you could do this:

https://stackoverflow.com/questions/767147/ignore-the-gitignore-file-itself/767213#767213